models/articles.model.js

```diff
@@ -63,5 +63,24 @@ exports.updateArticleVotesById = async (targetArticleId, incomingVotes) => {
             return article;
         }
     }
+}
+
+exports.selectCommentsByArticleId = async (targetArticleId) => {
+    if(!Number.isInteger(Number(targetArticleId))){
+        return Promise.reject({status:400, msg:'Bad request!'})
+    } else {
```

Comment:
no else needed here - if it falls into the if statement on line 69 the rest of the code won't run because it will return the rejection